Utils/test_upload_flow/create_test_branch.py

```diff
@@ -5,23 +5,25 @@
 import subprocess
 import time
 from pathlib import Path
-from typing import Union
 from git import GitCommandError, Head, Repo
 from zipfile import ZipFile
 from packaging.version import Version
 
 from Tests.scripts.utils import logging_wrapper as logging
 from Tests.scripts.utils.log_util import install_logging
+from Utils.github_workflow_scripts.utils import get_env_var
```

Comment:
how is it better than `getenv`, when a default is provided?

Reply:
Raises EnvVariableError If the environment variable is not set or empty and no default value is passed.

Reply:
Since here we do pass a default, I would use `os.getenv`